python-lib/type_conversion.py

```diff
@@ -0,0 +1,220 @@
+import datetime
+import logging
+import math
+import numpy as np
+import pandas as pd
+
+from tableauhyperapi import SqlType
+from tableauhyperapi import TypeTag
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO, format='Plugin: Tableau Hyper API | %(levelname)s - %(message)s')
+
+
+def to_dss_date(hyper_date):
```

Comment:
IS there a TimeZone component to the Hyper date?

Reply:
No, in Hyper there is TypeTag.DATE, TypeTag.TIMESTAMP and TypeTag.TIME whereas in DSS we only have one date storage type for all three [TypeTag.DATE](https://help.tableau.com/current/api/hyper_api/en-us/reference/py/tableauhyperapi.html#tableauhyperapi.TypeTag
)